module/.cypress/components/button.cy-test.tsx

```diff
@@ -0,0 +1,18 @@
+import { mount } from '@cypress/react';
+import { composeStories } from '@storybook/testing-react';
+import * as React from 'react';
+
+import * as stories from '../../src/components/button/button.stories';
+
+const { Default, Pending } = composeStories(stories);
+
+it('should render a default button', () => {
+  mount(<Default onClick={cy.stub().as('click')} />);
+  cy.get('button').should('have.text', Default.args?.children).click();
+  cy.get('@click').should('have.been.calledOnce');
```

Comment:
does the string arg have any kind of intellisense or autocomplete?

Reply:
In what sense? All of the matchers ('have.text' etc) have intellisense, but things like the focus of the .get() functions don't as they can literally be anything.